presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -681,6 +690,14 @@ DOUBLE_PRECISION
     : 'DOUBLE' WS 'PRECISION'
     ;
 
+INCLUDING_PROPERTIES
```

Comment:
Defining a token like this seems strange. I think we can do this like `NULLS FIRST` / `NULLS LAST` in `sortItem`